llama_stack/distribution/library_client.py

```diff
@@ -397,7 +400,10 @@ def _convert_body(self, path: str, method: str, body: dict | None = None) -> dic
         if not body:
             return {}
 
-        func, _, _ = find_matching_endpoint(method, path, self.endpoint_impls)
+        if self.route_impls is None:
```

Comment:
nit: this seems to be repeated a bunch across these methods. perhaps a helper function?